ArkKit/ark-render-kit/ArkCanvas.swift

```diff
@@ -0,0 +1,13 @@
+struct ArkCanvas: Canvas {
+    var canvasComponents: [CanvasComponent] = []
+
+    func render(using renderer: CanvasRenderer) {
+        for canvasComponent in canvasComponents {
+            canvasComponent.render(using: renderer)
```

Comment:
How should we support positioning of rendered components? And maintain sync with Physics side

Reply:
@chownces that should be under `ArkCanvasSystem`

- I added it as a `TODO`, will work there after physics is in.